### PR TITLE
Validate compile in multi-jvm

### PR DESCRIPTION
**Pull request by chbatey**
This should fail in travis

### CODE REVIEW

File: project/MultiNode.scala [hunk 1]
```diff
@@ -100,8 +109,7 @@ object MultiNode extends AutoPlugin {
       (headerCreate in MultiJvm).value
       (compile in MultiJvm).value
     }) ++ headerSettings(MultiJvm) ++ Seq(
-      // only works if I put it here ¯\_(ツ)_/¯
-      compile in MultiJvm := ((compile in MultiJvm).triggeredBy(compile in Test)).value
+      validateCompile := compile.?.all(anyConfigsInThisProject).value
     )
```
Comment (patriknw) [inline project/MultiNode.scala]: if we can't make it in `test:compile` we should add `validateCompile` to the new `all*` command aliases

Comment (chbatey) [reply project/MultiNode.scala]: :+1:

**Review by patriknw — APPROVED** [review]
LGTM

**Review by dwijnand — APPROVED** [review]
LGTM.  Land it, test it, then forward-fix or revert it after some shared usage.

**Comment by patriknw** [comment]
@chbatey fix conflict and merge this one

File: .travis.yml [hunk 2]
```diff
@@ -30,7 +30,7 @@ cache:
 script:
   - jabba use "adopt@~1.8.202-08"
   # need to override as the default is to test
-  - sbt -jvm-opts .jvmopts-travis ";++$TRAVIS_SCALA_VERSION update ;++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues ;++$TRAVIS_SCALA_VERSION test:compile"
+  - sbt -jvm-opts .jvmopts-travis ";++$TRAVIS_SCALA_VERSION update mimaReportBinaryIssues ;++$TRAVIS_SCALA_VERSION test:compile"
```
Comment (dwijnand) [inline .travis.yml]:
```suggestion
  - sbt -jvm-opts .jvmopts-travis ";++$TRAVIS_SCALA_VERSION update ;++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues ;++$TRAVIS_SCALA_VERSION test:compile ;++$TRAVIS_SCALA_VERSION validateCompile"
```

Lost an `;++$TRAVIS_SCALA_VERSION` and the `validateCompile`.

**Comment by chbatey** [comment]
Fixed conflict, can be merged after validation

**Pull request merged**